### ISSUE TITLE
Help with the simulator

### ISSUE THREAD

**Issue by ronea23** [timgabets/electron-atm#12]

**Comment by ronea23** [comment]
Hi Tim,

The App is pretty neat, applauds, so in order to use the ATM simulator do we need the ATM switch ? I see the simulator throws error 
![image](https://user-images.githubusercontent.com/10896354/60269548-f6194380-98ee-11e9-8b4e-c7e7deb0523a.png)

i assume this simulator needs some host simulator as well ?

**Comment by timgabets** [comment]
You're right, it needs atm host as wellTG16:20, June 27, 2019, ronea23 <notifications@github.com>:Hi Tim,
The App is pretty neat, applauds, so in order to use the ATM simulator do we need the ATM switch ? I see the simulator throws error

i assume this simulator needs some host simulator as well ?

**Comment by ronea23** [comment]
any ISO8583 HOST will be okay ?

**Comment by ronea23** [comment]
@timgabets  can i for the project ?

**Comment by timgabets** [comment]
No, it must be NDC-specific ATM hostTG16:27, June 27, 2019, ronea23 <notifications@github.com>:any ISO8583 HOST will be okay ?

**Comment by ronea23** [comment]
Oh okay, then something like, IST / B24 / Postillion should be connected...

**Comment by santakdalai90** [comment]
> No, it must be NDC-specific ATM hostTG16:27, June 27, 2019, ronea23 <notifications@github.com>:any ISO8583 HOST will be okay ? —You are receiving this because you commented.Reply to this email directly, view it on GitHub, or mute the thread.

Are there any open source/free NDC specific ATM hosts/simulators that I can use with electron-atm?
